### ISSUE TITLE
Does this plugin works on Linux or Mac ?

### ISSUE THREAD

**Issue by AvanteElectro** [Viladoman/CompileScore#35]
Hi !

Does this plugin works on Linux or Mac ? I don't see any .vsix to install it.

Cheers !

**Comment by Viladoman** [comment]
Hi! 

It is in my plan to provide something for Mac or Linux at some point, but we are not there yet. 

If you are using Clang, you could try the [SeeProfiler project](https://github.com/Viladoman/SeeProfiler) I did 3 years ago. It is an electron app that parses a directory looking for the .json files created by the clang -ftime-trace flag. 

I haven't checked that project in a while, it might need a pass.

**Comment by Trass3r** [comment]
I guess in theory it could work in a manual fashion.
Running the aggregator via wine (https://github.com/Viladoman/CompileScore/wiki/Score-Generation#process-whole-directory) and viewing it with the standalone tool on Windows or with Mono?

**Comment by jorgenpt** [comment]
If anyone's reading this in the future, there is currently [a Makefile](https://github.com/Viladoman/CompileScore/blob/master/DataExtractor/Makefile) in the `DataExtractor` folder that can be used to build a native Linux version of the DataExtractor.

**Comment by Viladoman** [comment]
Yes. I have been working lately to allow linux and mac users to use Compile Score. For now the Data Extractor should be buildable and runnable in those systems, but sadly there is still work to be done in order to browse the results too. The current plan is to at least have a standalone application and vscode extension to do so. But let's see how far I can get with it.
